.github/workflows/cpplint-scan.yml

```diff
@@ -0,0 +1,59 @@
+name: "Cpplint Scan"
+
+on:
+  push:
+    branches: [master, devel]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [master, devel]
+
+jobs:
+  cppcheck:
+    name: Cpplint
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    steps:
+      - name: "Checkout F´ Repository"
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: ./.github/actions/setup
+
+      - name: Install cpplint
+        run: pip install cpplint
+
+      - name: Install xsltproc
+        run: sudo apt-get install xsltproc -y
+
+      - name: Install sarif tool
+        run: npm install @microsoft/sarif-multitool
+
+      - name: Run cpplint & export output to cppcheck format
+        run: cpplint --counting=detailed --quiet --recursive . 2>&1 | python3 .github/scripts/cpplint_to_cppcheckxml.py &> cpplint_cppcheck_result.xml
+
+      - name: Convert cpplint results to SARIF
+        run: npx "@microsoft/sarif-multitool" convert "cpplint_cppcheck_result.xml" --tool "CppCheck" --output "cpplint_cppcheck_result.sarif" --force
```

Comment:
`--tool` should be CppLint, not CppCheck

Reply:
Ah I see, my bad, I thought this was just a way to define the tool name in the results as it appears in the Security tab on GitHub.